torch/optim/adamw.py

```diff
@@ -255,11 +297,27 @@ def adamw(
             [params, grads, exp_avgs, exp_avg_sqs, max_exp_avg_sqs, state_steps],
             differentiable=differentiable)
 
+    if fused is None:
```

Comment:
this would need to await #93184

Reply:
Yes, we can also re-use shared doc when things are more stable.